src/components/Header.js

```diff
@@ -10,9 +10,22 @@ const Header = () => (
             <div className="title">
               <h6 >ICO Transparency Monitor</h6>
             </div>
-            <p>A go to place for investors and journalists to learn whether a given ICO is transparent and
-                            can be trusted. The ICO Monitor serves as a transparency benchmark for the community to
-                            better evaluate offerings and make informed decisions based on objective metrics.</p>
+            <p>A go to place for investors and journalists to learn whether a given ICO is transparent and can be trusted.
+              The ICO Monitor serves as a transparency benchmark for the community to better evaluate offerings and make
+              informed decisions based on objective metrics.
+            </p>
+            <p>
+              We evaluate all ICOs based on their smart contracts.
+              <ul>
+                <li>Non-transparent: this rating means that one or more essential dimensions of trust and security receive an unfavorable rating.</li>
+                <li>Transparent with issues: this rating means that one or more non-essential dimensions of trust and security receive an unfavorable rating.</li>
+                <li>Transparent: this rating means that all dimension of trust and security receive a favorable rating.</li>
+              </ul>
```

Comment:
Please put link to this https://github.com/Neufund/ico-transparency-monitor#transparency-questionnaire
with text "Please read detailed explanation here"